src/components/sidenav/sidenav.js

```diff
@@ -244,16 +245,19 @@ function SidenavFocusDirective() {
  * @param {expression=} md-is-locked-open When this expression evaluates to true,
  * the sidenav 'locks open': it falls into the content's flow instead
  * of appearing over it. This overrides the `md-is-open` attribute.
```

Comment:
In the MD Spec's [Surface Behaviors](https://material.io/archive/guidelines/layout/responsive-ui.html#responsive-ui-surface-behaviors) when this value is `true`, it maps to `Permanent`. The default when this is `false` is `Temporary`. If you want to implement the `Persistant` behavior, then you need to allow the value of `md-is-locked-open` to be toggled like in this [CodePen example](https://codepen.io/Splaktar/pen/aPjBpV?editors=1010#0).

We should link to this part of the spec in the main SideNav docs (above) and mention these behavior names. We also need to have demos of each type of behavior (could be as part of 1 or 2 demos).